requirements-dev.txt

```diff
@@ -10,4 +10,5 @@ scanf>=1.4.1
 pytest
 pytest-cov
 coverage
-flake8
\ No newline at end of file
+flake8
+tox
```

Comment:
I've thought about this and this morning I had the idea of just gathering the requirements in `setup.py` from `requirements.txt` (and `-dev`). Apparently both have slightly different "features", in that `requirements.txt` is essentially a list of parameters to pass on to `pip`, so there's a few things that you can do there but not in `setup.py`.

In the end I decided that it might be a good idea to keep them around, if nothing else then for tools that want to install requirements without installing the package itself (IDE, or docker, see above?)